lib/rules/no-unknown-property.js

```diff
@@ -203,13 +204,12 @@ function getStandardName(name, context) {
   if (SVGDOM_ATTRIBUTE_NAMES[name]) {
     return SVGDOM_ATTRIBUTE_NAMES[name];
   }
-  let i = -1;
   const names = getDOMPropertyNames(context);
-  const found = names.some((element, index) => {
-    i = index;
-    return element.toLowerCase() === name;
-  });
-  return found ? names[i] : null;
+  // Let's find a possible attribute match with a case-insensitive search.
+  const foundName = names.find(
+    (element) => element.toLowerCase() === name.toLowerCase()
+  );
+  return foundName; // This can be undefined if the value hasn't been found.
```

Comment:
```suggestion
  return names.find(
    (element) => element.toLowerCase() === name.toLowerCase()
  );
```
the comment isn't really helpful since that's how `.find` already works :-)